src/import/importConfiguration.ts

```diff
@@ -153,6 +153,7 @@ export function importConfiguration(yaml: YAMLConfiguration | string, file: stri
     resources: parseResources(yaml.resources, file),
     pages: parsePages(yaml.pages, file),
     parameters: parseParameters(yaml, yaml.FSHOnly, file),
+    plugins: yaml.plugins,
```

Comment:
For some YAML fields we allow for "simple" declaration and complex declaration.  Do you think there is any value in supporting:
```yaml
plugins:
  - foo@1.0.0
```
as a simplified version of:
```yaml
plugins:
  - name: foo@1.0.0
```

Or do you think it would be better to be consistent?

Reply:
I kind of want to stick with only allowing the complex declaration, or at least requiring the complex declaration if the declaration includes arguments. Otherwise, I feel like there might be too many variations:
```yaml
plugins:
  - dog@1.0.0 # simple, name only
  - name: cat@1.0.3 # complex, name only
  - horse@3.1.1 # simple with args
    - potoooooooo
    - true
  - name: eel@4.1.1 # complex with args
    args:
    - electric
    - 3
```
Do four variations seem like too many?

Reply:
Ha! Yes! Four variations is too many! I guess I'm just trying to be consistent w/ how we treat other things in the config.  Here are some examples from the [exhaustive config](https://github.com/FHIR/sushi/pull/1065#discussion_r854394955) to consider:

```yaml
dependencies:
  hl7.fhir.us.core: 3.1.0
  hl7.fhir.us.mcode:
    id: mcode
    uri: http://hl7.org/fhir/us/mcode/ImplementationGuide/hl7.fhir.us.mcode
    version: 1.0.0

resources:
  Patient/my-example-patient:
    name: My Example Patient
    description: An example Patient
    exampleBoolean: true
  Patient/bad-example: omit

pages:
  index.md:
    title: Example Home
  implementation.xml:
  examples.xml:
    title: Examples Overview
    simpleExamples.xml:
    complexExamples.xml:
```

I'm noticing that none of our top-level properties are arrays.  They're all objects.  So I almost wonder if we should be considering this instead:

```yaml
plugins:
  dog: # simple name (local)
  cat: 1.0.3 # simple name and version (NPM)
  horse: # complex name and args (local)
    args:
    - potoooooooo
    - true
  eel: # complex name, version, args (NPM)
    version: 4.1.1
    args:
    - electric
    - 3
```

Ah... but now that I've typed that, I remember -- _order is significant_.  That's why it needs to be a sequence.  So... is this silly?

```yaml
plugins:
  - dog # simple name (local)
  - cat: 1.0.3 # simple name and version (NPM)
  - horse: # complex name and args (local)
      args:
      - potoooooooo
      - true
  - eel: # complex name, version, args (NPM)
      version: 4.1.1
      args:
      - electric
      - 3
```

I am probably overthinking it.  But since `dependencies` is `name: version`, it feels like maybe we should try to be at least a little consistent with that?  I dunno.